resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <b-container fluid>
+      <h3>
+        {{ $t('settings.application.title') }}
+        <b-button id="application-save-button"
+                  class="float-right"
+                  variant="success"
+                  @click="updateSettings(settings)"
+                  v-if="!viewOnly"
+                  :disabled="isBusy">
+          <span><i class="fas fa-save mr-2"></i>{{ $t('app.save') }}</span>
+        </b-button>
+      </h3>
+
+      <hr>
+
+    <!--Pagination page size settings-->
+    <b-form-group
+      label-for="application-name-input"
+      :description="$t('settings.application.name.description')"
+      :state='fieldState("name")'
+    >
+
+      <template v-slot:label>
+        {{ $t('settings.application.name.title') }}
```

Comment:
Why we need the extra template? The label could be set in the form group through an attribute!